Source/WebKit/UIProcess/Cocoa/ExtensionCapabilityGranter.mm

```diff
@@ -127,6 +127,13 @@ static bool finalizeGrant(const String& environmentIdentifier, AuxiliaryProcessP
     }
 
     if (grant.isValid()) {
+        auto& existingGrant = iterator->value;
+        ASSERT(!existingGrant.isValid());
+        if (existingGrant.isValid()) {
+            granterQueue().dispatchSync([&existingGrant]() mutable {
```

Comment:
Why do we need a dispatchSync() here? Can't we move the grant to the lambda and invalidate it async?

Reply:
That's a very good point, will fix!

Thanks for reviewing!